message_ix/model/MESSAGE-MACRO_run.gms

```diff
@@ -184,7 +184,9 @@ import_cost(node2, commodity, year) =
 * import into node2 from other nodes
           input(node2,tec,vintage,year,mode,node,commodity,level,time2,time)
         * duration_time_rel(time,time2) * ACT.L(node2,tec,vintage,year,mode,time2)
-        * COMMODITY_BALANCE.M(node,commodity,level,year,time) / discountfactor(year) )
+        * (COMMODITY_EQUIVALENCE.m(node,commodity,level,year,time) + 
```

Comment:
please format similar to the other GAMS code (space around mathematical brackets, indenting)